server/methods/core/cart.js

```diff
@@ -444,6 +448,18 @@ Meteor.methods({
     if (variant.weight || variant.height || variant.width || variant.length) {
       parcel = { weight: variant.weight, height: variant.height, width: variant.width, length: variant.length };
     }
+
+    // Use default settings for parcel weight, length, height, and width
```

Comment:
So these values should only be used when there is no parcel set on the product but it looks like these are going to be used in all cases. This needs to be wrapped in some sort of `if (!parcel)` conditional